posts/2022/05/thoughts-on-reviewing-for-accessibility.md

```diff
@@ -0,0 +1,185 @@
+<!--
+.. title: Thoughts on reviewing for accessibility: Maybe there is a checklist?
```

Comment:
I realize that this title is in conversation with the title of your previous post in the series ("[Not a checklist: different accessibility needs in JupyterLab](https://labs.quansight.org/blog/2021/09/not-a-checklist/)"), but to me, this title is a bit ponderous. Could we rework it to something that rolls off the tongue a bit more easily?

Something simple perhaps, like: My accessibility review checklist.

Or:

- A checklist? Some things to consider in an accessibility review
- Accessibility review: thoughts... and a checklist!
- "Checking" for accessibility (see what I did there?)